File: Visual Basic/PROJECTS/repos/RandomGenerator/RandomGenerator/Form1.vb
Public Class Form1

    Dim num As Integer

    Private Sub GetNumber()
        num = Math.Ceiling(Rnd() * 20)
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Randomize()
        GetNumber()
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Select Case (Val(TxtGuess.Text))
            Case Is > num
                LblMsg.Text = TxtGuess.Text & " is too high"
            Case Is < num
                LblMsg.Text = TxtGuess.Text & " is too low"
            Case Is = num
                LblMsg.Text = TxtGuess.Text & " is correct" &
                    "I have thought of another number - Try again!"
                GetNumber()
        End Select
        TxtGuess.Text = ""
    End Sub
End Class

File: Visual C#/SRC/Code/Entry/Form1.cs
namespace Entry
{
    public partial class Form1 : Form
    {
        private int num;
        public void setNum(int num) { this.num = num; }
        public int getNum() { return num; }

        public Form1()
        {
            InitializeComponent();
            Random rnd = new Random();
            setNum(rnd.Next(1, 21));
        }

        public void rate(int guess, int num) 
        {
            if (guess < num) label2.Text = "Too Low!";
            else
            if (guess > num) label2.Text = "Too High!";
            else
                label2.Text = "*** Correct ***";
        }

        private void button1_Click(object sender, EventArgs e)
        {
            int guess = Convert.ToInt16(textBox1.Text);
            rate(guess, getNum());
        }

      
    }
}